packages/debian/changelog

```diff
@@ -44,6 +44,9 @@ cgrates (0.11.0~dev) UNRELEASED; urgency=medium
   * [SessionS] Move *cost as subflag in *rals for
     SessionSv1.ProcessEvent
   * [DiameterAgent] Added DPR support
+  * [SupplierS] Add verification for event filters before populating
```

Comment:
Please rename to something like: Prefiltering ~*req. data sources to avoid unnecessary processing of costs/stats.